CraftTweaker2-API/src/main/java/crafttweaker/CraftTweakerAPI.java

```diff
@@ -217,6 +219,30 @@ public static void logError(String message, Throwable exception) {
     public static void logDefault(String message) {
         getLogger().logDefault(message);
     }
+
+    /**
+     * Set whether errors and warnings printed to players' log or not.
```

Comment:
When you say `players' log` do you mean their chat, or their actual log file?

If you mean the log file then:
```
Set whether errors and warnings are printed to the players' log or not.
```

If you mean their chat, then:
```
Set whether errors and warnings are printed to the players' chat or not.
```